File: src/main/java/com/epicdragonworld/gameserver/network/packets/sendable/CharacterDeletionResult.java
package com.epicdragonworld.gameserver.network.packets.sendable;

import com.epicdragonworld.gameserver.network.SendablePacket;

/**
 * @author Pantelis Andrianakis
 */
public class CharacterDeletionResult extends SendablePacket
{
	public CharacterDeletionResult()
	{
		// Send the data.
		writeShort(4); // Packet id.
	}
}

File: EpicDragonWorldServer/Network/Packets/Sendable/CharacterDeletionResult.cs
/**
 * Author: Pantelis Andrianakis
 * Date: November 7th 2018
 */
class CharacterDeletionResult : SendablePacket
{
    public CharacterDeletionResult()
    {
        // Send the data.
        WriteShort(4); // Packet id.
    }
}
